src/datachain/model/ultralytics/bbox.py

```diff
@@ -0,0 +1,160 @@
+"""
+This module contains the YOLO models.
```

Comment:
[Q] Does this need to be repeated in every file? Should this refer to the model in the file?